compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -481,6 +481,25 @@ TR::Instruction *fixedSeqMemAccess(TR::CodeGenerator *cg, TR::Node *node, intptr
          nibbles[idx] = cursor = generateMemSrc1Instruction(cg, opCode, node, memRef, srcOrTrg, cursor);
       }
 
+   // When using tempReg ensure no register spills occur in the middle of the fixed sequence
+   if (tempReg)
+      {
+      TR::RegisterDependencyConditions *dep;
+      if (srcOrTrg == tempReg)
```

Comment:
This code also needs to check for `srcOrTrg == baseReg`, otherwise it will create multiple dependencies for the same register in that case.

Reply:
Fixed in 218a9da